datahub-web-react/src/app/onboarding/utils.tsx

```diff
@@ -0,0 +1,48 @@
+import { Typography } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { StepStateResult } from '../../types.generated';
+import { onboardingConfig, OnboardingStep } from './onboarding';
+
+export function convertStepId(stepId: string, userUrn: string) {
+    const step = onboardingConfig.find((configStep) => configStep.id === stepId);
+    return `${userUrn}-${step?.id}`;
+}
+
+// used to get all of the steps on our initial fetch
+export function getStepIds(userUrn: string) {
+    return onboardingConfig.map((step) => `${userUrn}-${step.id}`);
+}
+
+const StepTitle = styled(Typography.Title)`
+    margin-botton: 5px;
+`;
+
+export function getStepsToRender(
```

Comment:
can we add some tests for this file?I'd say all three of these function should have some tests for them (and they should be pretty easy to write good unit tests for)

Reply:
I can address this in a follow up!